### PR TITLE
Fixes #6506: Change send_clean to push signature along with inventory

### PR DESCRIPTION
**Pull request by peckpeck**
http://www.rudder-project.org/redmine/issues/6506

Replacing previous PR: https://github.com/Normation/rudder-techniques/pull/653

### CODE REVIEW

**Comment by peckpeck** [comment]
@VinceMacBuche could you review this one ?

File: tools/send-clean.sh [hunk 1]
```diff
@@ -44,28 +46,32 @@ do
...
 
-# 3 - Send the file
-HTTP_CODE=$(${CURL_BINARY} --proxy '' -f -F file=@${FILENAME} -o /dev/null -w '%{http_code}' ${SERVER})
+# 4 - Send the file
+HTTP_CODE=$(${CURL_BINARY} --proxy '' -f -F "file=@${FILENAME}" ${SIGNATURE_OPT} -o /dev/null -w '%{http_code}' ${SERVER})
 SEND_COMMAND_RET=$?
 
-# 4 - Abort if sending failed
+# 5 - Abort if sending failed
 if [ ${SEND_COMMAND_RET} -eq 7 -o "${HTTP_CODE}" = "503" ]
```
Comment (VinceMacBuche) [inline tools/send-clean.sh]: Signature errors are 401 errors so you qkso have to check that error status

Comment (peckpeck) [inline tools/send-clean.sh]: Good catch

**Comment by peckpeck** [comment]
PR updated

**Comment by VinceMacBuche** [comment]
great! Merging!

**Pull request merged**